pkg/validation/config.go

```diff
@@ -298,6 +298,9 @@ func validateOperator(ctx *configContext, input *api.OperatorStepConfiguration,
 		if bundle.As == "" && bundle.BaseIndex != "" {
 			validationErrors = append(validationErrors, ctxN.AddField("base_index").errorf("base_index requires as to be set"))
 		}
+		if bundle.As == "" && bundle.SkipBuildingIndex {
+			validationErrors = append(validationErrors, ctxN.AddField("skip_building_index").errorf("skip_building_index requires as to be set"))
```

Comment:
```suggestion
			validationErrors = append(validationErrors, ctxN.AddField("skip_building_index").errorf("skip_building_index requires 'as' to be set"))
```

since 'as' is such a common word, it reads a little funny if we don't add quotes to signify it is a field name